Dockerfile-ubuntu-22.04

```diff
@@ -11,15 +11,12 @@ LABEL description="Docker Image for Building/Testing XLS on Ubuntu 22.04 x86-64"
 # Update package info
 RUN apt-get update -y
 
-# Install Bazel
-RUN apt-get install -y curl gnupg && \
-curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg && \
-mv bazel.gpg /etc/apt/trusted.gpg.d/ && \
-echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list && \
-apt-get update -y && apt-get install -y bazel
+# Install bazelisk from npm
+RUN apt-get install npm
+RUN npm install -g @bazel/bazelisk
```

Comment:
maybe just get the binary from the release page using curl/wget?
https://github.com/bazelbuild/bazelisk/releases

Reply:
Do we need to track the latest release, or just use the latest available version (i.e., v1.20) ?